### PR TITLE
Update EIP-3267: Fix SPDX license identifier

### PR DESCRIPTION
**Pull request by Pandapip1**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by Pandapip1** [comment]
Testing in prod moment

**Comment by Pandapip1** [comment]
Bump.

**Comment by lightclient** [comment]
@Pandapip1 I don't think we can unilaterally convert these licenses as the authors should approve the change.

**Comment by MicahZoltu** [comment]
I agree with @lightclient, these definitely need author approval.

**Comment by Pandapip1** [comment]
> @Pandapip1 I don't think we can unilaterally convert these licenses as the authors should approve the change.

Agreed. That's why I haven't been merging these w/o author approval, and have been letting @eth-bot handle the majority of the work.

**Comment by Pandapip1** [comment]
Still an issue.

File: assets/eip-3267/contracts/BaseRestorableSalary.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: LGPL-3.0-or-later
+// SPDX-License-Identifier: CC0-1.0
```
Comment (xinbenlv) [inline assets/eip-3267/contracts/BaseRestorableSalary.sol:1]: For licensing change like this, ask the original code author for approval?

Comment (Pandapip1) [reply assets/eip-3267/contracts/BaseRestorableSalary.sol:1]: Yes

**Comment by Pandapip1** [comment]
Waiting for author review.

**Comment by Pandapip1** [comment]
Waiting for author review.

**Comment by Pandapip1** [comment]
Waiting for review

**Comment by Pandapip1** [comment]
Waiting on author review

**Comment by vporton** [comment]
For the author everything seems OK.

**Comment by Pandapip1** [comment]
> For the author everything seems OK.

Great! Adding to manual merge queue.

**Pull request merged**